lib/runtime/reducers.js

```diff
@@ -288,14 +289,21 @@ reducers.sigma = {
                 } else if (n === 1) {
                     return 0;
                 } else {
-                    return math.sqrt(ops.mul(1/(n-1), ops.sub(ssum, ops.div(ops.mul(sum, sum), n))));
+                    return math.sqrt(ops.div(ops.sub(ssum, ops.div(ops.mul(sum, sum), n)), n));
                 }
             },
             update: function(pt) {
                 var v = pt[fld];
                 if (v !== undefined) {
-                    sum = ops.add(sum, v);
-                    ssum = ops.add(ssum, ops.mul(v, v));
+                    // Reduce error using the assumed mean method by shifting
+                    // the data based on the first value as suggested by:
+                    // https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
```

Comment:
Also, that link doesn't talk about the assumed mean, and the one that does, https://en.wikipedia.org/wiki/Assumed_mean, says
```
Its interest today is chiefly historical but it can be used to quickly estimate these statistics. There are other rapid calculation methods which are more suited for computers which also ensure more accurate results than the obvious methods.
```
which isn't exactly a ringing endorsement.